certified-connectors/ServiceDeskPlus Cloud/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6680 @@
+                          "description": "status -> name"
+                        },
+                        "id": {
+                          "type": "string",
+                          "title": "status.id",
+                          "description": "status -> id"
+                        }
+                      },
+                      "description": "status"
+                    },
+                    "scheduled_start_time": {
+                      "type": "object",
+                      "properties": {
+                        "display_value": {
+                          "type": "string",
+                          "title": "scheduled_start_time.display_value",
+                          "description": "scheduled_start_time -> display_value"
+                        },
+                        "value": {
+                          "type": "string",
+                          "title": "scheduled_start_time.value",
+                          "description": "scheduled_start_time -> value"
+                        }
+                      },
+                      "description": "scheduled_start_time"
+                    },
+                    "template": {
+                      "type": "object",
+                      "properties": {
+                        "inactive": {
+                          "type": "boolean",
+                          "title": "template.inactive",
+                          "description": "template -> inactive"
+                        },
+                        "name": {
+                          "type": "string",
+                          "title": "template.name",
+                          "description": "template -> name"
+                        },
+                        "id": {
+                          "type": "string",
+                          "title": "template.id",
+                          "description": "template -> id"
+                        }
+                      },
+                      "description": "template"
+                    },
+                    "has_release_association": {
+                      "type": "boolean",
+                      "description": "has_release_association"
+                    },
+                    "attachments": {
+                      "type": "array",
+                      "items": {},
+                      "description": "attachments"
+                    },
+                    "display_id": {
+                      "type": "object",
+                      "properties": {
+                        "display_value": {
+                          "type": "string",
+                          "title": "display_id.display_value",
+                          "description": "display_id -> display_value"
+                        },
+                        "value": {
+                          "type": "string",
+                          "title": "display_id.value",
+                          "description": "d
```

Comment:
@SDPCloud The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]. 
please use semicolon ";" to separate the categories.